CONTRIBUTING.md

```diff
@@ -151,6 +151,33 @@ Running the script for a project in `sdk\tables` would look like this:
 eng\scripts\Export-API.ps1 tables
 ```
 
+## Updating Sample Snippets
+If the specific client library has sample snippets in markdown format, they were most likely created with help of the `eng\scripts\Update-Snippets.ps1` script.
```

Comment:
Consider adding a reference to our [Samples guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples)  in case someone just stumbled here and wonders how to write samples in the first place